lib/diagnostic.js

```diff
@@ -38,19 +53,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Clifford',
+  species: 'dog',
+  breed: 'big red',
+  _vocalization: 'bark bark',
+  vocalize: function () {
+    return this._vocalization;
+  },
+
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
+const Pet = function (name, species, breed, vocalization) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this._vocalization = vocalization;
 };
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
-/* your answer here */
+Pet.prototype.vocalize = function () {
+  return this._vocalization;
+};
 
```

Comment:
Correct. If you try to run both with the previous code, you'll see the difference.